### ISSUE TITLE
System.Uri class is not serialisable

### ISSUE THREAD

**Issue by Porl91** [bridgedotnet/Bridge#2885]
In beta4 the `[Reflectable]` attribute was added to the `System.Uri` class, although it appears to be throwing an exception when I attempt to serialise a class that has a `System.Uri` as a property. Here's a quick repo:

https://deck.net/8a5ee9dfc27641cf91ce568d172bbb5f

Here's the exception that's thrown:

```js
System.Exception: ReferenceError: getAbsoluteUri is not defined
```
I originally posted this on issue #2769 although I split it out into a separate issue because that issue's more about making classes emit metadata rather than actually making them serialisable.

**Comment by DanTup** [comment]
We've hit this again today. We're having to convert Uris to strings which sucks a little. @geoffreymcgill currently this issue has no milestone - not sure if it may have been overlooked or is just low priority?

**Comment by LeonidVeriga** [comment]
The issue is  being fixed in **Bridge** **16.3.1** and **Bridge.Newtonsoft.Json** **1.2.1**
